nhr-loader/src/main/resources/NHRLoader.properties

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (C) 2021 B3Partners B.V.
+#
+# SPDX-License-Identifier: MIT
+#
+#
+
+app.version=${project.version} (git ${git.commit.id.describe})
+brmo.version=${project.version}
+app.user-agent=B3Partners NHR loader/${git.commit.id.describe}
+
+# CLI
+
+usage.headerHeading=B3Partners NHR loader (${git.commit.id.describe}) - (C) 2022 B3Partners B.V.%n
+usage.synopsisHeading=%nUsage:%n\u0020 
+usage.optionListHeading=%nOptions:%n
+usage.commandListHeading=%nCommands:%n
+nhr-loader.usage.footerHeading=%nUse a command with the --help option for information about the command, for example:%n  ${ROOT-COMMAND-NAME} request --help%n
```

Comment:
Ik zie liever Nederlandse teksten als default, brmo is een product is een specifiek p de NL markt gericht product

Reply:
Als je net als de bag2 en bgt loaders ook een NHRLoader_nl.properties toevoegt met Nederlandse teksten, krijg je met de Java optie `-Duser.language=nl` (of met systeem locale zoals bijv in Dockerfile `LANG=nl_NL.UTF-8 LANGUAGE=nl_NL:nl LC_ALL=nl_NL.UTF-8`) Nederlandse teksten bij runnen van de CLI.
In de brmo-service webapp krijg ik van de bag2/bgt loader ook Nederlandse teksten dus daar wordt de locale ook op NL gezet op een of andere manier.

Reply:
Ik heb een Nederlands messagebestand toegevoegd, zoals de BAG2 lader.